behaviours/meme_rater.py

```diff
@@ -276,10 +276,24 @@ async def delete_meme(event: hikari.GuildReactionAddEvent) -> None:
     if not shit_reaction:
         return
 
-    count = shit_reaction.count
+    shit_reaction_count = shit_reaction.count
+
+    # Find the "🔟" reaction; if not found then return 0.
+    ten_reaction = next(
+        (reaction for reaction in message.reactions if reaction.emoji == "🔟"), None
+    )
+
+    # Convert to int var
+    ten_reaction_count = ten_reaction.count if ten_reaction else 0
+
+    net_shit_count = (
+        shit_reaction_count - ten_reaction_count
+    )  # Final count of shit emojis offset by 10s.
+
     if not shit_reaction.is_me:
-        count += 1
-    if count >= int(os.getenv("MEME_VOTE_DELETE_THRESHOLD", 4)):
+        net_shit_count += 1
+
+    if net_shit_count >= int(os.getenv("MEME_VOTE_DELETE_THRESHOLD", 4)):
```

Comment:
can we make this ==

will hide the race condition :kekw:

Reply:
No.

Reply:
why?

Reply:
It will break things.

Reply:
This seems silly. >= makes sense here

Reply:
Why is the `>=` neccessary? In theory as soon as `==` is hit it's deleted

Reply:
Better to be safe than sorry

Reply:
Because there is no guarantee discord will send you a separate event for every reaction.

Reply:
I see. Alright merge it in VP of Kitti Software Development (VPoKSD) @adamnwerbik